src/main/scala/com/horizon/exchangeapi/ServicesRoutes.scala

```diff
@@ -259,6 +261,12 @@ trait ServicesRoutes extends JacksonSupport with AuthenticationSupport {
           "defaultValue": "bar"
         }
       ],
+     "secrets": [
+  "secret1": {
+    "type": "string",
+    "description": "string"
+ }
+],
```

Comment:
This is a minor thing, but please fix the indenting of this section so it is consistent with the others.